README.md

```diff
@@ -10,6 +10,21 @@ several services orchestrated by [Docker Compose](https://docs.docker.com/compos
 
 To get access to the GAE, [see the documentation on Slab](https://uclh.slab.com/posts/gae-access-7hkddxap)
 
+## Development
+
+[Follow the developer setup instructions](./docs/setup/developer.md).
+
+Before raising a PR, make sure to **run all tests** for each PIXL module
+and not just the component you have been working on as this will help us catch unintentional
+regressions without spending GitHub actions minutes :-)
```

Comment:
NGL I don't do this, as its a public project they're free 🤷🏻

Reply:
Yeah fair, I think this is leftover from when repo was still private